### PR TITLE
Feature/cms index filters

### PR DESCRIPTION
**Pull request by pjosh**

### CODE REVIEW

File: backend/app/controllers/backend/albums_controller.rb [hunk 1]
```diff
@@ -87,7 +87,7 @@ def set_album
       end
 
       def set_albums
-        @albums = Album.albums(@search, @index_items_limit * @page)
+        @albums = Album.albums(params, @index_items_limit * @page)
```
Comment (simaob) [inline backend/app/controllers/backend/albums_controller.rb]: let's use safe params for this. by creating a method that does something like:

`params.permit(:featured, :visibility_off)`

Makes it safer.

Comment (pjosh) [inline backend/app/controllers/backend/albums_controller.rb]: cool @simaob ! I've put it on the `ApplicationRecord` model.
If there is a better place to place it, please tell me!!

Comment (simaob) [inline backend/app/controllers/backend/albums_controller.rb]: Normally that goes in the controller. So the ApplicationController of the backend is probably the best way, as it's shared by other controllers.

Comment (simaob) [inline backend/app/controllers/backend/albums_controller.rb]: It's similar to what we do here: https://github.com/Vizzuality/grid-arendal/blob/master/backend/app/controllers/backend/albums_controller.rb#L80

Comment (pjosh) [inline backend/app/controllers/backend/albums_controller.rb]: thx @simaob ! Good to go!!

**Pull request merged**